zebra-rpc/src/methods.rs

```diff
@@ -694,6 +709,61 @@ where
         }
         .boxed()
     }
+
+    fn get_address_utxos(&self, addresses: Vec<String>) -> BoxFuture<Result<Vec<GetAddressUtxos>>> {
+        let mut state = self.state.clone();
+        let mut response_utxos = vec![];
+
+        async move {
+            // TODO: make this a function after #4119 is merged
```

Comment:
Optional: #4119 got merged a few hours ago, did you want to make this change now, or in another PR?

Thanks for picking this up, seems like a good refactor for all the address methods.